### ISSUE TITLE
maybe move some work to applicationDidBecomeActive

### ISSUE THREAD

**Issue by r10s** [deltachat/deltachat-ios#745]
[`didFinishLaunchingWithOptions`](https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1622921-application) is always called, independently of whether the app is started for a short background fetch or started "full" with ui etc.

imu, maybe someone can confirm that, after `didFinishLaunchingWithOptions`, apple calls 
- either [`applicationDidBecomeActive`]( https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1622956-applicationdidbecomeactive) - to become a foreground app with ui etc.
- or [`performFetchWithCompletionHandler`](https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1623125-application)

having that in mind, maybe too much work is done in `didFinishLaunchingWithOptions `, things that are needed only for the ui can go to `applicationDidBecomeActive`. also wondering if that isn't the better place for UIWindow creation, cc @nayooti 

in other words, i think, `didFinishLaunchingWithOptions` should not do much more that what is also needed for `performFetchWithCompletionHandler`

_nb: nothing actionable right now, but i'd like to improve that on the async-move #740  :)_

**Comment by nayooti** [comment]
I'll look into this. 
What I can say now that it is normal to set window in `didFinishLaunchingWithOptions`. For iOS13-Apps this has changed btw.

**Comment by r10s** [comment]
thanks for input, then we maybe better leave UIWindow in didFinishLaunchingWithOptions, the [UIWindow docs](https://developer.apple.com/documentation/uikit/uiwindow) also says "Windows do not have any visual appearance of their own".

**Comment by r10s** [comment]
closing this for now, we'll refactor when we're on it.
